src/backend/main/py.main/caliopen_main/objects/base.py

```diff
@@ -264,7 +265,14 @@ def marshall_db(self, **options):
                         setattr(self._db, att,
                                 getattr(self, att).replace(tzinfo=None))
                     else:
-                        setattr(self._db, att, getattr(self, att))
+                        if issubclass(self._attrs[att], CaliopenObject):
+                            if getattr(self, att) is not None:
```

Comment:
too complicated to read, use correctly named variables instead of this chain of getattr